### PR TITLE
Fixed passthrough chat for 1.19

### LINKED ISSUE #57: 1.19 support: Chat spoofing is not supported on this client version error - change passthrough implementation
Chat spoofing was quietly deprecated in BungeeCord for Minecraft 1.19, meaning the plugin spits out an exception on that platform when using passthrough chat on v1.19+. (https://github.com/SpigotMC/BungeeCord/issues/3327).
Instead on relying on `ProxiedPlayer#chat()` to pass through chat to backends, the event should instead just not be cancelled (unless it doesn't satisfy a filter's requirements). Downside of this of course is that it means chat replacers (emoji) won't be able to be passed - but that's a reasonable concession imo.
Velocity (and maybe Bungee) will probably provide API (Velocity might have already added this actually) letting us do a "chat preview", should also add support for this when that API gets added so players can see how their message will appear.

### PR DESCRIPTION
**Pull request by TrueWinter**
Closes #57.

### CODE REVIEW

File: common/src/main/resources/languages/en-gb.yml [hunk 1]
```diff
@@ -43,4 +43,5 @@ error_reply_none_online: '[Error:](#ff3300) [Nobody in the last group of people
 error_last_message_not_group: '[Error:](#ff3300) [The last message you received was not a group message.](#ff7e5e)'
 error_no_messages_opt_out: '[Error:](#ff3300) [You have not yet sent or received group messages.](#ff7e5e)'
 removed_from_group_message: '[Removed you from the group message between:](#00fb9a) %1%'
-list_conjunction: 'and'
\ No newline at end of file
+list_conjunction: 'and'
+passthrough_shortcut_command_error: '[Error:](#ff3300) [Sending messages to passthrough channels using shortcut commands is not supported. Please switch to the channel first.](#ff7e5e)'
```
Comment (WiIIiam278) [inline common/src/main/resources/languages/en-gb.yml:47]: I was trying to think of a better wording for this locale, but I don't have one. Worth mentioning that chat spoofing does work in Velocity still

File: velocity/src/main/java/net/william278/huskchat/velocity/player/VelocityPlayer.java [hunk 2]
```diff
@@ -60,11 +60,6 @@ public boolean hasPermission(String permission) {
         return player.hasPermission(permission);
     }
 
-    @Override
-    public void passThroughChat(String message) {
-        player.spoofChatInput(message);
```
Comment (WiIIiam278) [inline velocity/src/main/java/net/william278/huskchat/velocity/player/VelocityPlayer.java:65]: This still works in 1.19, actually. I'm not sure we want to degrade functionality on Velocity for our Bungee users to be honest.

**Review by WiIIiam278 — CHANGES_REQUESTED** [review]
It may be a bit more clunky but I think we should still support sending messages through shortcut commands on velocity. Perhaps make passThroughChat throw an exception (PassThroughNotSupportedException) on Bungee and catch that exception with the user error locale, while letting it run fine on Velocity.

File: common/src/main/resources/languages/en-gb.yml (hunk 1 above)
Comment (TrueWinter) [reply common/src/main/resources/languages/en-gb.yml:47]: Yeah, I can't think of better wording for this either.

**Pull request merged**